Dockerfile

```diff
@@ -19,16 +19,35 @@ WORKDIR node-red-contrib-security-nodes
 RUN npm install && npm link agile-sdk && npm link
 WORKDIR ..
 
-COPY agile-node-red-nodes agile-node-red-nodes
-RUN npm install agile-node-red-nodes
+ARG NODE=agile-node-red-nodes
+COPY $NODE $NODE
+RUN npm install $NODE
 
 # adding Xively support
-COPY node-red-contrib-agile-xively node-red-contrib-agile-xively
-RUN npm install node-red-contrib-agile-xively
+ARG NODE=node-red-contrib-agile-xively 
+COPY $NODE $NODE
+RUN npm install $NODE
 
 # adding FIWARE support
-COPY node-red-contrib-agile-fiware node-red-contrib-agile-fiware
-RUN npm install node-red-contrib-agile-fiware
+ARG NODE=node-red-contrib-agile-fiware
+COPY $NODE $NODE
+RUN npm install $NODE
+
+ARG NODE=node-red-contrib-agile-solid
+COPY $NODE $NODE
+RUN npm install $NODE
+
+ARG NODE=node-red-contrib-agile-owncloud
+COPY $NODE $NODE
+RUN npm install $NODE
+
+ARG NODE=node-red-contrib-agile-googledrive
+COPY $NODE $NODE
+RUN npm install $NODE
+
+ARG NODE=node-red-contrib-agile-thingspeak
+COPY $NODE $NODE
+RUN npm install $NODE
```

Comment:
This Dockerfile only installs agile-node-red-nodes and node-red-contrib-security-nodes. I will try to check why.